recipes/docopt.cpp/0.6.2/test_package/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class DoctestTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def configure(self):
+        if tools.cross_building(self.settings):
```

Comment:
why do we need this in test package?

Reply:
We needed this in our local recipes, but since conan-center does not cross-build anything (yet?), it is useless right now

Reply:
However, if it is supported one day, every test_package will most likely break

Reply:
I am not sure why does it break in test package. may be there is some conan bug? do you remember what was an error if not using it?

Reply:
It does not break per se, but you cannot run the executable you just built, which will fail the test

Reply:
which is why I simply assert that the executable exists.

Reply:
no, I am asking about deleting `compiler.libcxx` setting for test_package. we never needed it for test package in our practice, so I am curious.

Reply:
As you are building C++ app, you should not remove libcxx. Is there a specific limitation or bug?

Reply:
This is an antique leftover for when having this setting with Android broke CMake integration, will remove